### ISSUE TITLE
Overlap function

### ISSUE THREAD

**Issue by rockdesignnl** [somewebmedia/hc-offcanvas-nav#93]
Hi! I was wondering if it was possible to overlap containers without pushing the layer underneath to the right. So that no mather how deep you go you stay in same container width.

Any idea on this?

Cheers!

**Comment by Danishnaseer** [comment]
Yes you can do that by changing the position of the nav-menu to absolate then fixed, in this case the menu will push open but the side widht of the container will be remain same
